integration-tests/olminstall/suite/cluster_api_health.py

```
@@ -0,0 +1,188 @@
+def openshift_console_route_unavailable_reason(*, probe: bool = True) -> str:
+    """Return infra reason when the cluster console route hostname is unreachable (EaaS lease DNS)."""
+    if not probe:
+        return ""
+    try:
+        result = oc_run(
+            ["whoami", "--show-console"],
+            check=False,
+            capture_output=True,
+            timeout=15,
+        )
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)